edx_lint/files/pylintrc

```diff
@@ -36,28 +249,93 @@ load-plugins=edx_lint.pylint,pylint_django,pylint_celery
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
+    # These should be left to the discretion of the reviewer
+    bad-continuation,
+    invalid-name,
+    misplaced-comparison-constant,
+    too-many-lines,
+    file-ignored,
+    too-many-nested-blocks,
+    too-many-statements,
```

Comment:
I think this should default to enable; it calls out overly-complex code that can yield more bugs and maintainability problems.